Promact.Oauth.Server/src/Promact.Oauth.Server/project.json

```diff
@@ -1,116 +0,0 @@
-      "version": "1.0.0-preview4-final",
-      "type": "build"
-    },
-    "Microsoft.VisualStudio.Web.CodeGeneration.Tools": {
-      "version": "1.0.0-preview4-final",
-      "type": "build"
-    },
-    "Microsoft.VisualStudio.Web.CodeGenerators.Mvc": {
-      "version": "1.0.0-preview4-final",
-      "type": "build"
-    },
-    "System.Net.Mail": "1.0.0-rtm-00002",
-    "NLog.Extensions.Logging": "1.0.0-*",
-    "SendGrid.NetCore": "1.0.0-rtm-00002",
-    "BundlerMinifier.Core": "2.2.306",
-    "Microsoft.AspNetCore.Server.IISIntegration": "1.1.0",
-    "Microsoft.VisualStudio.Web.BrowserLink.Loader": "14.1.0",
-    "MimeKit": "1.10.0",
-    "NLog.Config": "4.4.1",
-    "System.Collections": "4.3.0",
-    "AutoMapper": "5.2.0",
-    "MailKit": "1.10.1",
-    "Microsoft.AspNetCore.Authentication.Cookies": "1.1.0",
-    "Microsoft.AspNetCore.Diagnostics": "1.1.0",
-    "Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore": "1.1.0",
-    "Microsoft.AspNetCore.Identity.EntityFrameworkCore": "1.1.0",
-    "Microsoft.AspNetCore.Mvc": "1.1.0",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0",
-    "Microsoft.AspNetCore.StaticFiles": "1.1.0",
-    "Microsoft.EntityFrameworkCore.SqlServer": "1.1.0",
-    "Microsoft.EntityFrameworkCore.SqlServer.Design": "1.1.0",
-    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.1.0",
-    "Microsoft.Extensions.Configuration.Json": "1.1.0",
-    "Microsoft.Extensions.Configuration.UserSecrets": "1.1.0",
-    "Microsoft.Extensions.Logging": "1.1.0",
-    "Microsoft.Extensions.Logging.Console": "1.1.0",
-    "Microsoft.Extensions.Logging.Debug": "1.1.0",
-    "Microsoft.Extensions.Options.ConfigurationExtensions": "1.1.0",
-    "Exceptionless.AspNetCore": "4.0.1927",
-    "Microsoft.IdentityModel.Protocols": "2.1.2",
-    "coveralls.net": "0.8.0-unstable0013",
-    "OpenCover": "4.6.519",
-    "System.Linq": "4.3.0",
-    "System.Linq.Parallel": "4.3.0",
-    "microsoft.aspnetcore.server.kestrel.https": "1.1.0"
-  },
-
-  "tools": {
-    "Microsoft.AspNetCore.Razor.Tools": "1.1.0-preview4-final",
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.1.0-preview4-final",
-    "Microsoft.EntityFrameworkCore.Tools": "1.1.0-preview4-final",
-    "Microsoft.Extensions.SecretManager.Tools": "1.1.0-preview4-final",
-    "Microsoft.VisualStudio.Web.CodeGeneration.Tools": {
-      "version": "1.1.0-preview4-final",
-      "imports": [
-        "portable-net45+win8"
-      ]
-    }
-  },
-
-  "frameworks": {
-    "netcoreapp1.1": {
-      "dependencies": {
-        "Microsoft.NETCore.App": {
-          "version": "1.1.0",
-          "type": "platform"
-        }
-      },
-      "imports": [
-        "dotnet5.6",
-        "portable-net45+win8"
-      ]
-    }
-  },
-
-  "buildOptions": {
-    "emitEntryPoint": true,
-    "preserveCompilationContext": true
-  },
-
-  "runtimeOptions": {
-    "configProperties": {
-      "System.GC.Server": true
-    }
-  },
-
-  "publishOptions": {
```

Comment:
In .csproj

<ItemGroup>
    <None Update="wwwroot\**\*;Views\**\*;Areas\**\Views;nlog.config;Template\**\*">
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </None>
  </ItemGroup>

the above given code has been generated.